include/databroker/DataBrokerConfiguration.hpp

```diff
@@ -30,24 +30,51 @@
 namespace eprosima {
 namespace databroker {
 
-struct DataBrokerConfiguration
+constexpr const char* DEFAULT_CONFIGURATION_FILE = "DATABROKER_CONFIGURATION.yaml";
+constexpr const char* DEFAULT_PRIVATE_KEY_FILE = "db.key";
+constexpr const char* DEFAULT_DH_FILE = "dbparam.pem";
+constexpr const char* DEFAULT_CA_FILE = "ca.crt";
+constexpr const char* DEFAULT_CERT_FILE = "db.crt";
```

Comment:
I think this is never gonna be used. But we can keep it for now

Reply:
It would be used in case the `--tls` option is set as argument, in order to avoid adding a different argument for each file